pkg/storagedistribution/storagedistribution.go

```diff
@@ -118,10 +118,17 @@ func AddDisk(
 		}
 	}
 
+	maxDriveCount := dm.Rows[0].InstanceMaxDrives
+	for _, row := range dm.Rows {
```

Comment:
This may not be true for all instance types. Can you add a note here for future so we remember to modify this when we have specific instance types in the the rows?

Reply:
For each instance types we will have a separate row and a corresponding max drive count limit.
So we will need this even in that case right ?

Reply:
Basically we will have a drive count limit (openstorage/PX enforce) or (cloud) enforced.